boardgame-ui/src/components/activities/ActivityBox.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div v-if=activity class="activity-container">
+        <article class="media" v-if="activity.type === 'event/join'">
+            <div class="media-left calendar-media-left">
+                <figure class="is-64x64"><span><i class="fa-3x far fa-calendar-alt"></i></span></figure>
+            </div>
+            <div class="media-content">
+                {{$t("activity.event.join")}} <strong>{{activity.event.name}}</strong>.
+                <br/>
+                <p class="activity-datetime">{{activity.datetime}}</p>
```

Comment:
We could use moment.js for more user-friendly date display (but not needed for now I think ^^)

Reply:
Indeed, but I think we should do more: like develop few functions for displaying dates throughout the application. We need this because the API returns all datetimes in UTC so we need to convert them to the user timezone. I could work on that if you agree on the idea.

Reply:
Don't you think http://momentjs.com/ and http://momentjs.com/timezone/ cover our needs?

Reply:
We should use this indeed, but I don't want for instance the format string to be repeated everywhere. I think it would be great to have a small function on top of those.

Reply:
I created a new component bgc-datetime to encapsulate datetime formatting !